docs/usage.rst

```diff
@@ -55,6 +55,97 @@ Participants file: <bids-root>/participants.tsv
 Either file should include age (or if you wish to be more explicit: age_months) columns, and it is recommended to have an accompanying JSON file to further describe these fields, and explicitly state the values are in months.
 
 
+**************************
+Preparing data for QSIPrep
+**************************
+
+QSIPrep is a BIDS App, meaning that it expects the data to be in BIDS format.
+However, QSIPrep does contain some idiosyncrasies that mean that the data may need to be prepared in a specific way.
+
+
+Siemens Reverse Phase-Encoded "Field Maps"
+==========================================
+
+Reverse phase-encoded images are a common acquisition for distortion correction.
+For a dMRI scan, this would mean acquiring one or more volumes of b=0 images with the
+opposite phase encoding direction of the main dMRI scan.
+
+Unlike Philips and GE, Siemens will not let you acquire a scan that only contains b=0 volumes,
+so researchers often acquire a short dMRI run with a mix of b=0 and b>0 volumes.
+QSIPrep expects these short scans to be in the fmap directory, instead of the dwi directory.
+If you acquire data like this, you should organize your data as below::
+
+    sub-<label>/
+        ses-<label>/
+            dwi/
+                sub-<label>_ses-<label>_dir-AP_dwi.nii.gz
+                sub-<label>_ses-<label>_dir-AP_dwi.bval
+                sub-<label>_ses-<label>_dir-AP_dwi.bvec
+                sub-<label>_ses-<label>_dir-AP_dwi.json
+            fmap/
+                sub-<label>_ses-<label>_dir-PA_epi.nii.gz
+                sub-<label>_ses-<label>_dir-PA_epi.bval
+                sub-<label>_ses-<label>_dir-PA_epi.bvec
+                sub-<label>_ses-<label>_dir-PA_epi.json  # Add IntendedFor field here
+
+In this scenario, the short scan is organized as a field map, with the epi suffix.
+As of BIDS v1.10.0, EPI field maps can have bval and bvec files, so this organization is completely BIDS-compliant.
+
+If you organize your short scan as a dMRI run, QSIPrep will denoise the short scan and concatenate it with the longer run,
+which is not optimal.
+
+
+Complex-Valued Data
+===================
+
+If you acquire complex-valued data, you should (1) split the data into magnitude and phase files (NOT real and imaginary!)
+and (2) retain a copy of the bval and bvec files with the part-mag entity included (i.e., do not use the inheritance principle on these files).
```

Comment:
Doesn't this conflict with below where you need to have files with both the `part-mag` and the `part-phase`

Reply:
You need them for each file for BIDS (e.g., with both `part-mag` and `part-phase` or without `part` at all), but you need it with `part-mag` for QSIPrep.

So this is BIDS-valid:

```
sub-01_dwi.bval
sub-01_dwi.bvec
sub-01_part-mag_dwi.nii.gz
sub-01_part-phase_dwi.nii.gz
```

But on the other hand you need this for QSIPrep:

```
sub-01_part-mag_dwi.bval
sub-01_part-mag_dwi.bvec
sub-01_part-mag_dwi.nii.gz
sub-01_part-phase_dwi.nii.gz
```

Note that the part-phase bval and bvec are not needed for QSIPrep.

Reply:
I'll make changes to make this clearer